src/components/Plans/UpdatePaymentInformation/UpdatePaymentInformation/ContactInformation/index.js

```diff
@@ -0,0 +1,175 @@
+                              className="field-item--50"
+                            />
+                          </FieldGroup>
+                        </FieldItem>
+                        <FieldItem className="field-item">
+                          <FieldGroup>
+                            <PhoneFieldItem
+                              fieldName={fieldNames.phone}
+                              id="phone"
+                              label={`*${_('updatePaymentMethod.payment_method.transfer.phone')}`}
+                              placeholder={_('forms.placeholder_phone')}
+                              className="field-item--50 dp-p-r"
+                              required
+                            />
+                            <EmailFieldItem
+                              type="text"
+                              fieldName={fieldNames.email}
+                              id="email"
+                              label={`*${_('updatePaymentMethod.payment_method.transfer.email')}`}
+                              className="field-item--50"
+                              required
+                            />
+                          </FieldGroup>
+                        </FieldItem>
+                        {status !== SAVED && (
+                          <FieldItem className="field-item">
+                            <div className="dp-buttons-actions">
+                              <SubmitButton className="dp-button button-medium primary-green">
+                                {_('updatePaymentMethod.payment_method.transfer.send_button')}
+                              </SubmitButton>
+                            </div>
+                          </FieldItem>
+                        )}
+                        {showMessage && (
+                          <FieldItem className="field-item">
+                            <div
+                              className={`dp-wrap-message dp-wrap-${
+                                status === SAVED ? 'success' : 'cancel'
+                              }`}
+                              role="alert"
+                              aria-label={status === SAVED ? 'success' : 'cancel'}
+                            >
+                              <span className="dp-message-icon" />
+                              <div className="dp-content-message">
+                                <p>
+                                  {_(
+                                    status === SAVED
+                                      ? 'updatePaymentMethod.payment_method.transfer.send_email_success_message'
+                                      : 'updatePaymentMethod.payment_method.transfer.send_email_error_message',
+                                  )}
+                                </p>
+                              </div>
+                            </div>
+                            <hr className="m-t-24 m-b-24" />
+                          </FieldItem>
+                        )}
```

Comment:
What do you think about create a component with this code and test it separately?